exporter/kafkaexporter/README.md

```diff
@@ -12,10 +12,25 @@ The following settings are required:
 The following settings can be optionally configured:
 - `brokers` (default = localhost:9092): The list of kafka brokers
 - `topic` (default = otlp_spans): The name of the kafka topic to export to
-- `metadata.full` (default = true): Whether to maintain a full set of metadata. 
+- `metadata`
+  - `full` (default = true): Whether to maintain a full set of metadata. 
                                     When disabled the client does not make the initial request to broker at the startup.
-- `metadata.retry.max` (default = 3): The number of retries to get metadata
-- `metadata.retry.backoff` (default = 250ms): How long to wait between metadata retries
+  - `retry`
+    - `max` (default = 3): The number of retries to get metadata
+    - `backoff` (default = 250ms): How long to wait between metadata retries
+- `timeout` (default = 5s): Is the timeout for every attempt to send data to the backend.
+- `retry_on_failure`
+  - `enabled` (default = true)
+  - `initial_interval` (default = 5s): Time to wait after the first failure before retrying; ignored if `enabled` is `false`
```

Comment:
Are the default values correct? Shouldn't we use the same defaults as the driver uses? 

https://github.com/Shopify/sarama/blob/master/config.go#L462

Reply:
You pointed to the timeout this is the initial backoff delay after a request fail. We can tune the timeout to be 10sec if that is important, I would leave it as is for the moment and maybe collect feedback.